src/Components/JoinScreen/Chatbox.tsx

```diff
@@ -0,0 +1,61 @@
+import { ChangeEvent, ReactElement, useEffect, useState } from "react";
+
+import { Button } from "@mui/material";
+import socket from "../Websocket/socket";
+import { useParams } from "react-router-dom";
+
+const Chatbox = (): ReactElement => {
+  const {roomName} = useParams()
+  const [messages, setMessages] = useState<string[]>([])
```

Comment:
messages should probably be an object. We should probably make a message type
```
type Message= {
  author: string,
  message: string,
  time: Date
}
```
The time property is maybe not needed? The idea is that if it somehow gets scrambled, we can still sort the messages and display them properly. Maybe if multiple people send a message at the same time, this could resolve it?

Later on, we could probably consider a "to" property and a "room" property. The "to" property would be for private chats if we decide to do that. The room property may or may not be needed, I'm thinking if we do that, we no longer need to send the roomName in the socket call. Maybe could use it for other stuff in the future.